libs/interpret-text/src/lib/TextExplanationDashboard/Control/TextExplanationView/TextExplanationView.tsx

```diff
@@ -148,19 +148,23 @@ export class TextExplanationView extends React.PureComponent<
                   weightLabels={this.props.weightLabels}
                 />
               </Stack.Item>
-              <Stack.Item id="TextChoiceGroup">
-                <ChoiceGroup
-                  defaultSelectedKey="all"
-                  options={options}
-                  onChange={this.changeRadioButton}
-                  required
-                />
-              </Stack.Item>
-              <Stack.Item>
-                <Text variant={"small"}>
-                  {localization.InterpretText.View.legendText}
-                </Text>
-              </Stack.Item>
+              {this.props.selectedWeightVector !== WeightVectors.AbsAvg && (
+                <Stack.Item id="TextChoiceGroup">
+                  <ChoiceGroup
+                    defaultSelectedKey="all"
+                    options={options}
+                    onChange={this.changeRadioButton}
+                    required
+                  />
+                </Stack.Item>
+              )}
+              {this.props.selectedWeightVector !== WeightVectors.AbsAvg && (
+                <Stack.Item>
+                  <Text variant={"small"}>
+                    {localization.InterpretText.View.legendText}
+                  </Text>
+                </Stack.Item>
+              )}
```

Comment:
just curious, why not just do this?
```
{this.props.selectedWeightVector !== WeightVectors.AbsAvg && (
                <Stack.Item id="TextChoiceGroup">
                  <ChoiceGroup
                    defaultSelectedKey="all"
                    options={options}
                    onChange={this.changeRadioButton}
                    required
                  />
                </Stack.Item>
                <Stack.Item>
                  <Text variant={"small"}>
                    {localization.InterpretText.View.legendText}
                  </Text>
                </Stack.Item>
              )}
```

Reply:
I can't put two <Stack.Item> without nesting them, typescript does not allow it.  However, putting them in a single div will break this parent stack/child stack.item code, and creating another stack is not desirable here.